homeassistant/components/husqvarna_automower/lawn_mower.py

```diff
@@ -61,7 +71,7 @@ async def async_setup_entry(
         "override_schedule",
         {
             vol.Required("override_mode"): vol.In(OVERRIDE_MODES),
-            vol.Required("duration"): vol.All(
+            vol.Required(CONF_DURATION): vol.All(
```

Comment:
These are attributes, so they should be prefixed with `ATTR_*`

Reply:
done in https://github.com/home-assistant/core/pull/126311/commits/c19b8498cbcee638758a99eb1e5dae787775c1c6
I also changed it for the days. Hope that's okay.